can't resolve `es6.object.get-own-property-symbols`

version 1.0.1

When babel config is 
```
    "presets": [
      [
        "env",
        {
          "targets": {
            "browsers": [
              "last 2 versions"
            ]
          },
          "debug": true,
          "modules": false,
          "useBuiltIns": true
        }
      ],
      "react"
    ],
```
i get the error
```
ERROR in ./app/app.js
Module not found: Error: Can't resolve 'core-js/modules/es6.object.get-own-property-symbols' in '/Users/andykenward/Documents/_github/bob/app'
 @ ./app/app.js 32:0-61
```

[travis-ci log](https://travis-ci.org/andykenward/bob/builds/182858583)
[repo commit](https://github.com/andykenward/bob/tree/df63fd05a8bd24453833c40f4a2d0a16b791882b)

--- a/data/builtInFeatures.js
+++ b/data/builtInFeatures.js
@@ -38,6 +38,7 @@ const es2015 = {
   "es6.symbol": {
     features: [
       "Symbol",
+      "Object static methods / Object.getOwnPropertySymbols",
       "well-known symbols / Symbol.hasInstance",
       "well-known symbols / Symbol.isConcatSpreadable",
       "well-known symbols / Symbol.iterator",
@@ -54,7 +55,6 @@ const es2015 = {
 
   "es6.object.assign": "Object static methods / Object.assign",
   "es6.object.is": "Object static methods / Object.is",
-  "es6.object.get-own-property-symbols": "Object static methods / Object.getOwnPropertySymbols",
   "es6.object.set-prototype-of": "Object static methods / Object.setPrototypeOf",
 
   "es6.function.name": 'function "name" property',

--- a/data/builtIns.json
+++ b/data/builtIns.json
@@ -262,15 +262,6 @@
     "android": 4.1,
     "ios": 9
   },
-  "es6.object.get-own-property-symbols": {
-    "chrome": 38,
-    "opera": 25,
-    "edge": 12,
-    "firefox": 36,
-    "safari": 9,
-    "node": 0.12,
-    "ios": 9
-  },
   "es6.object.set-prototype-of": {
     "chrome": 34,
     "opera": 21,

--- a/test/fixtures/preset-options/use-builtins-regenerator-true/expected.js
+++ b/test/fixtures/preset-options/use-builtins-regenerator-true/expected.js
@@ -29,7 +29,6 @@ import "core-js/modules/es6.promise";
 import "core-js/modules/es6.symbol";
 import "core-js/modules/es6.object.assign";
 import "core-js/modules/es6.object.is";
-import "core-js/modules/es6.object.get-own-property-symbols";
 import "core-js/modules/es6.object.set-prototype-of";
 import "core-js/modules/es6.function.name";
 import "core-js/modules/es6.string.raw";

